SourceX/miniwin/ddraw.h

```diff
@@ -6,7 +6,7 @@
 
 namespace dvl {
 
-extern int refreshDelay;
+extern double refreshDelay;

```

Comment:
Instead of using floating point math, consider changing the units to nanoseconds

Reply:
Done